index.html

```diff
@@ -100,5 +100,20 @@ <h3 class="content_area2">Why Choose Us</h3>
 
       </form>
     </section>
+
+    <section>
```

Comment:
Please, I think there is no need for the section as long as the footer element is used to use a proper semantic format.